doc/modules/grid_search.rst

```diff
@@ -139,6 +140,22 @@ For continuous parameters, such as ``C`` above, it is important to specify
 a continuous distribution to take full advantage of the randomization. This way,
 increasing ``n_iter`` will always lead to a finer search.
 
+There is also a log-uniform random variable with
+:class:`utils.fixes.lograndom`, which is continuous version of ``[1, 10, 100,
+1000]`` and are all equally likely with a log-uniform random variable.
```

Comment:
The wording needs improvement. Maybe,
```
There is also a log-uniform random variable (
:class:`utils.fixes.lograndom`) which is continuous version of ``[1, 10, 100,
1000]``.
```
as I'm not sure how to reasonably word the end of the sentence (but feel free to try).